temporalnexus/operation.go

```diff
@@ -340,13 +384,19 @@ func ExecuteUntypedWorkflow[R any](
 		internal.SetRequestIDOnStartWorkflowOptions(&startWorkflowOptions, nexusOptions.RequestID)
 	}
 
+	var encodedToken string
 	if nexusOptions.CallbackURL != "" {
 		if nexusOptions.CallbackHeader == nil {
 			nexusOptions.CallbackHeader = make(nexus.Header)
 		}
-		if idHeader := nexusOptions.CallbackHeader.Get(nexus.HeaderOperationID); idHeader == "" {
-			nexusOptions.CallbackHeader.Set(nexus.HeaderOperationID, startWorkflowOptions.ID)
+		encodedToken, err = generateWorkflowRunOperationToken(nctx.Namespace, startWorkflowOptions.ID)
```

Comment:
What if the `encodedToken` exceeds the server limit, but `startWorkflowOptions.ID` doesn't? So the workflow gets started but the start request cannot succeed?

Reply:
Good point but we should just bump the server limits for the token.